inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -148,23 +181,27 @@ struct jit_uni_roi_pooling_kernel_f32 : public jit_uni_roi_pooling_kernel, publi
                     }
                 }
 
-                add(aux_reg_input1, jpp_.c_block * sizeof(float));
+                add(aux_reg_input1, jpp_.c_block * jpp_.src_data_size);
 
                 inc(w_iter);
                 cmp(w_iter, reg_kw);
                 jl(w_loop_label, T_NEAR);
             }
 
-            add(aux_reg_input, jpp_.iw * jpp_.c_block * sizeof(float));
+            add(aux_reg_input, jpp_.iw * jpp_.c_block * jpp_.src_data_size);
 
             inc(h_iter);
             cmp(h_iter, reg_kh);
             jl(h_loop_label, T_NEAR);
         }
 
+        int dst_c_off = jpp_.oh * jpp_.ow * jpp_.c_block * jpp_.dst_data_size;
         for (int i = 0; i < c_blocks; i++) {
             Vmm vmm_dst = get_acc_reg(i);
-            uni_vmovups(ptr[reg_output + i * jpp_.oh * jpp_.ow * jpp_.c_block * sizeof(float)], vmm_dst);
+
+            store_emitter->emit_code({static_cast<size_t>(vmm_dst.getIdx())}, {static_cast<size_t>(reg_output.getIdx())},
+                                     std::make_shared<store_emitter_context>(Precision::FP32, jpp_.dst_prc, step, i * dst_c_off),
+                                     {store_pool_vec_idxs}, {store_pool_gpr_idxs});
```

Comment:
Seems that we do not need to instantiate the temporal copies of `store_pool_vec_idxs` and `store_pool_vec_idxs`. Here and below.

Reply:
Done